errors/error.go

```diff
@@ -22,36 +22,39 @@ type Category int
 type Reason int
 
 const (
-	// Success indicates no error occurred.
+// Success indicates no error occurred.
 	Success Category = 1000 * iota // 0XXX
 
-	// CertificateError indicates a fault in a certificate.
+// CertificateError indicates a fault in a certificate.
 	CertificateError // 1XXX
 
-	// PrivateKeyError indicates a fault in a private key.
+// PrivateKeyError indicates a fault in a private key.
 	PrivateKeyError // 2XXX
 
-	// IntermediatesError indicates a fault in an intermediate.
+// IntermediatesError indicates a fault in an intermediate.
 	IntermediatesError // 3XXX
 
-	// RootError indicates a fault in a root.
+// RootError indicates a fault in a root.
 	RootError // 4XXX
 
-	// PolicyError indicates an error arising from a malformed or
-	// non-existent policy, or a breach of policy.
+// PolicyError indicates an error arising from a malformed or
+// non-existent policy, or a breach of policy.
 	PolicyError // 5XXX
 
-	// DialError indicates a network fault.
+// DialError indicates a network fault.
 	DialError // 6XXX
 
-	// APIClientError indicates a problem with the API client.
+// APIClientError indicates a problem with the API client.
 	APIClientError // 7XXX
 
-	// OCSPError indicates a problem with OCSP signing
+// OCSPError indicates a problem with OCSP signing
 	OCSPError // 8XXX
 
-	// CSRError indicates a problem with CSR parsing
+// CSRError indicates a problem with CSR parsing
 	CSRError // 9XXX
+
+// CTError indicates a problem with the certificate transparency process
+	CTError // 10XXX
```

Comment:
This should also be added to [`/doc/errorcode.txt`](https://github.com/cloudflare/cfssl/blob/master/doc/errorcode.txt).